packages/twenty-server/src/engine/api/graphql/workspace-schema-builder/factories/connection-type-definition.factory.ts

```diff
@@ -43,7 +44,25 @@ export class ConnectionTypeDefinitionFactory {
     objectMetadata: ObjectMetadataInterface,
     options: WorkspaceBuildSchemaOptions,
   ): GraphQLFieldConfigMap<any, any> {
-    const fields: GraphQLFieldConfigMap<any, any> = {};
+    const fields: GraphQLFieldConfigMap<any, any> = Object.assign(
+      {},
+      ...getAvailableAggregationsFromObjectFields(objectMetadata.fields).map(
```

Comment:
I think we should follow the existing factory pattern here:
this.connectionTypeFactory.create(...) see lines below